### PR TITLE
[Ready] Syndicate Forgotten Ship ruin. Balance+Bug fix.

### LINKED ISSUE #50117: Cybersun hardsuits have no cold protection
Issue reported from  Round ID: [132654](https://scrubby.melonmesa.com/round/132654) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1507
and you can't turn on the temp stabilizer

### LINKED ISSUE #50122: Mapping error on new syndie space ruin
## Reproduction:
image

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Fixes problem with Cybersun hardsuits not having thermo-regulation button to protect from cold/hot environments.
Adds more useful stuff to the ship and cargo pod.

Fixes #50122
Fixes #50117

## Why It's Good For The Game

Bugs aren't good for health and having circuit boards with no parts to finish machines is lame!

## Changelog
:cl: EgorDinamit
add: Forgotten ship ruin now additionally contains: Deluxe stock parts box, box with beakers, 3 toolboxes, box with basic firing pins and a little bit more of resources in a protected vault and so on, find it out!
add: Replaces interrogation room with medbay.
fix: Fixed cybersun hardsuits by adding thermo-regulation button to it.
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Moment, I'll add some more stuff due to popular requests.

**Comment by Tharcoonvagh** [comment]
Close #50118?

**Review by Rohesie — DISMISSED** [review]
Code is fine.

**Comment by Tharcoonvagh** [comment]
Also you sure bout this:
Screenshot_20200320_210107_com android chrome

**Comment by EgorDinamit** [comment]
> Also you sure bout this:
> Screenshot_20200320_210107_com android chrome

Yeah, of course, I guess.

**Comment by RaveRadbury** [comment]
https://github.com/tgstation/tgstation/issues/50122

Could you take care of this extra reinforced table while you're at it

**Comment by EgorDinamit** [comment]
> #50122
> 
> Could you take care of this extra reinforced table while you're at it

Huh, okay, I'll fix it.

**Comment by ishitbyabullet** [comment]
> 
> 
> #50122
> 
> Could you take care of this extra reinforced table while you're at it

holy shit i was gonna mention that. thanks, bro.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -651,7 +651,7 @@
 	icon_state = "sleeper_s"
 	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
-	important_info = "Obey orders given by your captain. DO NOT leave the ship."
+	important_info = "Obey orders given by your captain. DO NOT let ship fall into enemy's hands."
```
Comment (Rohesie) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy's hands."
```

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -673,7 +673,7 @@
 	icon_state = "sleeper_s"
 	short_desc = "You are the captain of an old ship, stuck in hostile space."
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Command your crew and turn your ship into the most protected fortress."
-	important_info = "Protect the ship and secret documents in your backpack with your own life. DO NOT leave the ship."
+	important_info = "Protect the ship and secret documents in your backpack with your own life. DO NOT let ship fall into enemy's hands."
```
Comment (Rohesie) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
	important_info = "Protect the ship and secret documents in your backpack with your own life. DO NOT let the ship fall into enemy hands."
```

**Comment by EgorDinamit** [comment]
Not ready yet...

**Comment by EgorDinamit** [comment]
Now it is ready.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 3]
```diff
@@ -651,7 +651,7 @@
 	icon_state = "sleeper_s"
 	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
-	important_info = "Obey orders given by your captain. DO NOT let ship fall into enemy's hands."
+	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy's hands."
```
Comment (Rohesie) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy hands."
```
Or `the enemy's`
Same for the other one.

Comment (EgorDinamit) [reply code/game/objects/structures/ghost_role_spawners.dm]: Already did this, isn't it?

**Review by Rohesie — DISMISSED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 4]
```diff
@@ -673,13 +673,16 @@
...
+	important_info = "Protect the ship and secret documents in your backpack with your own life. DO NOT let the ship fall into enemy hands."
 	mob_gender = "male"
 	outfit = /datum/outfit/syndicatespace/syndicaptain
 
 /datum/outfit/syndicatespace/syndicaptain/post_equip(mob/living/carbon/human/H)
 	H.faction |= ROLE_SYNDICATE
 
+/obj/effect/mob_spawn/human/syndicatespace/captain/Destroy()
+	new/obj/structure/fluff/empty_sleeper/syndicate/captain(get_turf(src))
+
```
Comment (Rohesie) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
/obj/effect/mob_spawn/human/syndicatespace/captain/Destroy()
	new/obj/structure/fluff/empty_sleeper/syndicate/captain(get_turf(src))
	return ..()

```

**Pull request merged**